include/osgAnimation/RigGeometry

```diff
@@ -57,17 +56,17 @@ namespace osgAnimation
 
         META_Object(osgAnimation, RigGeometry);
 
-        inline void setInfluenceMap(VertexInfluenceMap* vertexInfluenceMap) { _vertexInfluenceMap = vertexInfluenceMap; }
-        inline const VertexInfluenceMap* getInfluenceMap() const { return _vertexInfluenceMap.get();}
-        inline VertexInfluenceMap* getInfluenceMap() { return _vertexInfluenceMap.get();}
+        inline void setInfluenceMap(VertexInfluenceMap* vertexInfluenceMap)  _vertexInfluenceMap = vertexInfluenceMap; }
```

Comment:
It looks codeblock has removed the { completely from this inline method, and shouldn't compile any more.  Could you double check it?